README.md

```diff
@@ -1,4 +1,6 @@
-**Earth Enterprise** is the open source release of Google Earth Enterprise, a geospatial application which provides the ability to build and host custom 3D globes and 2D maps.  Earth Enterprise does not provide a private version of Google imagery that's currently available in Google Maps or Earth.
+# Earth Enterprise [![Slack](https://img.shields.io/badge/chat-slack-ff69b4.svg)](http://slack.opengee.org) [![Join the chat at https://gitter.im/earthenterprise/Lobby](https://badges.gitter.im/earthenterprise/Lobby.svg)](https://gitter.im/earthenterprise/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I think it also makes sense for the text to be consistent. I think you should change the link to: https://img.shields.io/badge/chat-on%20slack-ff69b4.svg